pkg/crc/podman/podman.go

```diff
@@ -0,0 +1,82 @@
+package podman
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	crcos "github.com/code-ready/crc/pkg/os"
+)
+
+const (
+	rootless = "crc"
+	rootful  = "crc-root"
+)
+
+type Config struct {
+	PodmanExecutablePath string
+}
+
+func NewConfig() Config {
+	return Config{
+		PodmanExecutablePath: filepath.Join(constants.CrcOcBinDir, constants.PodmanRemoteExecutableName),
+	}
+}
```

Comment:
At this point, `Config` is not needed, whatever needs to execute podman can build the path as required (or this can be a global variable in this module if you prefer). This should make the code simpler.